common/entity/Person.ts

```diff
@@ -66,6 +67,16 @@ export abstract class Person {
         default: false,
     })
     isRedacted: boolean;
+
+    @Column({
+        type: 'timestamp',
+        nullable: true,
+        default: new Date(0),
+    })
+    lastTimeCheckedNotifications: Date;
+
+    @Column({ nullable: true, type: 'json' })
+    notificationPreferences: Prisma.JsonValue;
```

Comment:
@Jonasdoubleyou is Prisma.JsonValue the correct type?

Reply:
No, NotificationPreferences was better.

Reply:
The entity/* Files are from TypeORM, maybe better not to mix in Prisma Types here